src/components/TodoItem.js

```diff
@@ -1,9 +1,55 @@
-import React from "react"
-
-class TodoItem extends React.Component {
-  render() {
-    return <li>{this.props.todo.title}</li>
-  }
-}
-
-export default TodoItem
\ No newline at end of file
+/* eslint-disable no-console */
+/* eslint-disable react/state-in-constructor */
+/* eslint-disable react/no-unused-state */
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/button-has-type */
```

Comment:
- [ ] Kindly avoid disabling linters because they guide us in writing code that is clean and well-formatted.